backend/.env.test

```
@@ -0,0 +1,8 @@
+PORT="3011"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)